paper.md

```diff
@@ -35,38 +35,55 @@ a host of other tools within the JAX ecosystem.
 [example notebooks](https://github.com/CQCL/qujax/tree/main/examples).
 
 
-`qujax` represents a quantum circuit by a collection of three native Python iterables: a series of gate
-identifiers (either a string pointing to an object in `qujax.gates`, a unitary array or a function that
-outputs a unitary array), a series indicating which qubits to apply the gates to and finally a series
-of parameter indices indicating where, if any, to extract parameters for said gate from a single
-parameter vector.
+`qujax` represents a quantum circuit as a collection of three equal length Python iterables: 
+* a series of gate identifiers specifying the sequence of quantum gates to be applied to the qubits as part of the circuit. Each element can be either 
+  * a string referring to a gate in `qujax.gates` e.g. `"Z"`, `"Rx"`
+  * a JAX array representing a unitary matrix
+  * a function that returns such an array
+* a series indicating which qubits in the circuit each gate should be applied to 
+* a series of indices indicating which entries of a parameter vector (which is provided when later evaluating the circuit) correspond to parameters of the gate (e.g. rotation gates such as `"Rx"` take one parameter).
+
+A valid quantum circuit specification would be, for instance, `["X", "Rz", "Rz", "CRz"], [[0], [1], [0], [0, 1]], [[], [0], [1], [1]]` (TODO: add pictorial representation of circuit here).
```

Comment:
picture representation is a good idea! We can just insert the output from `qujax.print_circuit` either as a string or screenshot